### ISSUE TITLE
n2p2, The nnp potential of three elements ( eg : Cu, S, C ), can be used in MD simulations of two elements ?(eg: Cu2S)

### ISSUE THREAD

**Issue by YK-KK** [CompPhysVienna/n2p2#190]
The nnp potential of three elements ( eg : H, O, C ), can be used in MD simulations of two elements ? ( eg : H2O ), If so, what should be noted. Thank you.

**Comment by owen-rett** [comment]
Yes, I've done so with a NNP trained on Zr, Ce, O and using it for pure Zr, O. The only thing you have to be wary of is setting the emap correctly. For example, this would be using a NNP trained on Zr, Ce, O in a simulation where the lammps atom types are 1 for Zr, and 2 for O. 

`pair_style nnp dir ${nnpDir} showew yes showewsum 100 resetew no maxew 100 cflength 1.0 cfenergy 1.0 emap "1:Zr, 2:O"`
